include/config.h

```diff
@@ -31,6 +31,7 @@ struct kanshi_profile_output {
 
 struct kanshi_profile {
 	struct wl_list link;
+	char* name;
```

Comment:
Style: `*` on the right

Reply:
Fixed in the next commit. I can rebase if you really prefer.

Reply:
Yeah I'd prefer to keep commits separate and logically split